src/projects/digital/site/src/containers/DigitalItemNav/index.tsx

```diff
@@ -40,21 +40,27 @@ export const DigitalItemNav = () => {
         );
     });
 
+    const updateICs = () => setState({
```

Comment:
put it inside the useEffect otherwise you have to add it as a dependency and wrap it in a useCallback and all the other react BS (might not have to necessarily with react compiler now but still i would be safe, especially since `circuit` CAN change now)